gatsby-source-graphcms/README.md

```diff
@@ -1 +1,105 @@
 # `gatsby-source-graphcms`
+
+> A Gatsby source plugin for GraphCMS projects 🚀
+
+## Installation
+
+```shell
+yarn add gatsby-source-graphcms
+```
+
+## Configuration
+
+> We recommend using environment variables with your GraphCMS token and endpoint values. You can learn more about using environment variables with gatsby [here](https://www.gatsbyjs.org/docs/environment-variables/).
+
+### Basic
+
+```js
+// gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: 'gatsby-source-graphcms',
+      options: {
+        endpoint: process.env.GRAPHCMS_ENDPOINT,
+      },
+    },
+  ],
+}
+```
+
+### Authorization
+
+You can also provide an auth token using the `token` configuration key. This is necessary if your GraphCMS project is **not** publicly available, or you want to scope access to a specific content stage (i.e. draft content).
+
+```js
+// gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: 'gatsby-source-graphcms',
+      options: {
+        endpoint: process.env.GRAPHCMS_ENDPOINT,
+        token: process.env.GRAPHCMS_TOKEN,
+      },
+    },
+  ],
+}
+```
+
+### Options
+
+- `endpoint` _String_ (**required**)
+
+  - The endpoint URL for the GraphCMS project. This can be find in the [project settings UI](https://graphcms.com/docs/guides/concepts/apis#working-with-apis).
```

Comment:
```suggestion
  - The endpoint URL for the GraphCMS project. This can be found in the [project settings UI](https://graphcms.com/docs/guides/concepts/apis#working-with-apis).
```